### DISCUSSION TITLE
Mathjs help site

### DISCUSSION THREAD

**Discussion by dvd101x** [josdejong/mathjs#3274] [Show and tell]
Hi, just to share an explorer for mathjs help object.

It includes:

- quick search
- navigation by categories
- evaluate examples

https://github.com/dvd101x/mathjs-help

![imagen](https://github.com/user-attachments/assets/5afa906f-56ba-4615-82cd-40c5becb30ee)

This is work in progress and the intention is to include it in a development environment like [Engineering-Solver](https://github.com/dvd101x/Engineering-Solver) and to explore some ideas of implementing something like [docsify](https://docsify.js.org) on the official mathjs docs.

**Comment by josdejong** [comment]
That's nice! It would be nice to replace the Google Search of the website with a better search and interactive docs like this :)
